sarra/sr_xattr.py

```diff
@@ -127,6 +127,10 @@ def __init__(self, path):
                 #   unlikely to be a difficulty in real life? but perhaps worthy of an error message.
                 pass
 
+        if 'integrity' in self.x:
```

Comment:
same as before, in sr_message.py. Not sure if it matters

Reply:
that one is correct. it's to "upgrade" attributes when written by a previous version. convert it to identity
within the code.